docs/chain-rel-notes.md

```diff
@@ -4,22 +4,55 @@ title: "Chain Release Notes"
 
 # Chain Release Notes
 
-## Release notes for testnet 1.1.1
+## Testnet 1.2.1-pre-release 
 
-- A new feature, called **consensus-based weights** will be released soon in the testchain 1.1.1 only. Currently while calculating the dividends to a subnet validator, a quantity called instantaneous bond value is used. A subnet validator maintains an instantaneous bond value on each subnet miner. Instead of directly using the instantaneous bond value, a moving average, weighted over the current epoch and several previous epochs, is used. See the below equation for how this moving average is computed.
+The below are the advanced release notes. **The testnet 1.2.1-pre-release will be available on 26th June 2024.**
+
+### New features
+
+#### Consensus-based weights
+
+A new feature, called **consensus-based weights** is available in the testnet 1.2.1-pre-release. Currently while calculating the dividends to a subnet validator, a quantity called exponential moving average bond value of a subnet validator, weighted over the current epoch and several previous epochs, is used. See the below equation for how this moving average is computed.
 
 $$
 B_{ij}^{(t)} = \alpha\cdot\Delta B_{ij}^{(t)} + (1-\alpha)\cdot B_{ij}^{(t-1)}
 $$
 
-:::tip what changed
-Currently the $\alpha$ is set to `0.9`. With the consensus-based weights feature, this $\alpha$ value is made a variable. In the current version of this feature this change occurs behind-the-scenes, i.e., no specific user action is enabled. 
+Until now, the value of $\alpha$ is set to `0.9`. With this new feature, the $\alpha$ value is made into a variable. An optimium value for the variable $\alpha$ is determined based on the current consensus in a given subnet. Hence this feature is called **consensus-based weights**. A subnet owner can now experiment for the optimum value of $\alpha$. See the below documentation links for how to use this feature:
```

Comment:
Should we include the `why` i.e. making weights more speculative , combined with commit reveal , it rewards early movers?

Reply:
Good point. I will add it to the doc first, then see if I can add it to the rel notes without making rel notes too complicated.